### DISCUSSION TITLE
[V2]Paste image without domain

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-tiptap-editor#174] [Q&A]
Hello, i use curator MediaAction, and it past image with full url. When using many stages for testing content and data, its very uncomfortable to work with that. Because every time i need to replace domain with new domain. Looks like tiptap still can work if image has src like this `/storage/media/example.jpg`
Is there any reason to use full url? and its possible without write new Action class change this behavior?)

**Comment by awcodes** [comment]
I'll explore some with v3. Not really looking to add new functionality to v2 at the moment. I'm open to PR's though.

If I remember correctly, there was an issue with some conditionals around the storage disk that made it need the full url, but I'm not sure.
